components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -78,23 +100,24 @@ void HommeDynamics::set_grids (const std::shared_ptr<const GridsManager> grids_m
 
   // Outputs
   FieldLayout dyn_scalar_3d_mid_layout { dyn_3d_scalar_state_tags,  {ne, NTL,    NGP,NGP,NVL} };
+  FieldLayout dyn_scalar_3d_int_layout { dyn_3d_scalar_state_tags,  {ne, NTL,    NGP,NGP,NVL+1} };
   FieldLayout dyn_vector_3d_mid_layout { dyn_3d_vector_state_tags,  {ne, NTL,  2,NGP,NGP,NVL} };
   FieldLayout dyn_tracers_layout       { dyn_3d_tracer_state_tags,  {ne,QNTL,QSZ,NGP,NGP,NVL} };
-  m_computed_fields.emplace("v",  dyn_vector_3d_mid_layout, m/s, dyn_grid_name);
-  m_computed_fields.emplace("t",  dyn_scalar_3d_mid_layout, K,   dyn_grid_name);
-  m_computed_fields.emplace("dp", dyn_scalar_3d_mid_layout, Pa,  dyn_grid_name);
-  m_computed_fields.emplace("qdp",dyn_tracers_layout,       Qdp, dyn_grid_name);
-
-  const int ftype = get_homme_param_value<int>("ftype");
+  m_computed_fields.emplace("u",       dyn_vector_3d_mid_layout, m/s, dyn_grid_name);
+  m_computed_fields.emplace("vtheta",  dyn_scalar_3d_mid_layout, K,   dyn_grid_name);
+  m_computed_fields.emplace("phi",     dyn_scalar_3d_int_layout, Pa*pow(m,3)/kg, dyn_grid_name);
+  m_computed_fields.emplace("w",       dyn_scalar_3d_int_layout, m/s, dyn_grid_name);
+  m_computed_fields.emplace("dp",      dyn_scalar_3d_mid_layout, Pa,  dyn_grid_name);
+  m_computed_fields.emplace("qdp",     dyn_tracers_layout,       Qdp, dyn_grid_name);
```

Comment:
I'm guessing that we plan to add the physics tendencies as a field later when we hook up physics + dynamics?

Reply:
Yes, I haven't really put much effort into adding all the fields that Dynamics will need. My primary goal was to have homme compiled in, and working. AD hooks can be adjusted later.

Reply:
Okay cool.  That is what I figured, but wanted to make sure I mentioned it.  We may want to create a git issue as well so we can keep track of the big ToDo items.